src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_LateBindingHelpers.vb

```diff
@@ -990,11 +986,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' </summary>
         Private Function TryGetSpecialMember(Of T As Symbol)(<Out> ByRef result As T,
                                                        memberId As SpecialMember,
-                                                       syntax As SyntaxNode) As Boolean
+                                                       syntax As SyntaxNode,
+                                                       Optional isOptional As Boolean = False) As Boolean
 
             result = Nothing
             Dim useSiteInfo As UseSiteInfo(Of AssemblySymbol) = Nothing
-            Dim memberSymbol = Binder.GetSpecialTypeMember(Me._topMethod.ContainingAssembly, memberId, useSiteInfo)
+            Dim memberSymbol = Binder.GetSpecialTypeMember(Me._topMethod.ContainingAssembly, memberId, useSiteInfo, isOptional)
```

Comment:
>isOptional [](http://example.com/codeflow?start=116&length=10)

I suggest simply following the pattern used in the function above and reverting all changes made in `Binder.vb`.